xla/stream_executor/gpu/gpu_timer.cc

```diff
@@ -52,12 +52,18 @@ absl::Duration RandomDuration() {
 }
 
 bool ShouldLaunchDelayKernel() {
+#if GOOGLE_CUDA
```

Comment:
Maybe this is fine. But if we could use a normal "if" that would be even better. Not sure if we have a runtime variable somewhere around here, which we could query for cuda vs rocm?

Reply:
sure, let me think about it.. Maybe I can get access to device_description of StreamExecutor to check it at runtime

Reply:
not relevant anymore..